docs/section12/Rule12-4.md

```diff
@@ -0,0 +1,49 @@
+# Receptacle - Rule 12-4
+**Schema Version** 0.0.37  
+**Primary Rule:** True  
+**Rule ID:** 12-4  
+**Rule Description:** Computer room equipment schedules shall be modeled as a constant fraction of the peak design load per the following monthly schedule: Months 1, 5, 9 — 25%; Months 2, 6, 10 — 50%; Months 3, 7, 11 — 75%; Months 4, 8, 12 — 100%   
+**Appendix G Section:** G3.1.3.16   
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**
+1. Computer rooms are present in the baseline model  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Miscellaneous Equipment object in the baseline model    
+**Data Lookup:** None  
+**Function Call:**
+1) find_all
+2) is_space_a_computer_room
+3) compare_schedules
+
+## Applicability Checks:  
+- Iterate through the spaces in the baseline RMD: `for space_b in find_all("$.buildings[*].building_segments[*].zones[*].spaces[*]", B_RMD):`
+  - Check if the space is a computer room: `if is_space_a_computer_room(space_b):`
+    - Iterate through the miscelleneous equipment loads: `for misc_equip_b in space_b.miscellaneous_equipment:`
+      - Check if the equipment is IT equipment: `if misc_equip_b.type == "INFORMATION_TECHNOLOGY_EQUIPMENT" and misc_equip_b.power > 0:`
```

Comment:
@JacksonJ-KC do we need to check the misc equipment type? is the space type check and equip power >0 sufficient?
Just concerned about a scenario where the mis equip type might not be set- is that likely?

Reply:
My thought in doing in this way was that even a computer room might have regular plug loads that are not "computer room equipment". I am glad you brought this up though because I was thinking this would be good to get additional input on

Reply:
Based on the definition in 90.1 I don't think we need to check the equipment type. 

computer room: a room whose primary function is to house equipment for the processing
and storage of electronic data and that has a design electronic data equipment power density
exceeding 20 W/ft2 of conditioned floor area.

G3.1.3.16. _Computer room equipment_ schedules shall be modeled as a constant fraction of the
peak design load per the following monthly schedule:

Because it does not differentiate the different types of equipment in a computer room I think it is safe to apply the schedule fraction requirement to all the equipment in the room.  Note that computer room and equipment are two separate defined terms. There is no computer room equipment definition.

Reply:
The space type check includes power check - so you will be fine just using the `is_space_a_computer_room` for this applicability check.

Reply:
This seems like an area where semantics could change our approach. 

Is "Computer room equipment schedules" referring to schedules for computer room equipment, or is it referring to equipment schedules for computer rooms?

In my eyes, it seems reasonable to have computer room equipment, or INFORMATION_TECHNOLOGY_EQUIPMENT, as a separate MiscellaneousEquipment object from the typical plug loads for the space and have them on independent schedules.

Reply:
@weilixu in `is_space_a_computer_room` it seems like if the lighting space type is COMPUTER_ROOM then the power check is bypassed

Reply:
Jackson, see my comment. I think if you look at 90.1 language it means the equipment (all of it) in a computer room. You disagree with my logic?

Reply:
I see your argument Christina but I am not sure it is a slam dunk. Hoping to get input from more people on this and go with the majority consensus

Reply:
> @weilixu in `is_space_a_computer_room` it seems like if the lighting space type is COMPUTER_ROOM then the power check is bypassed

This function might need to be revisited - it basically checks for one or the other - lighting or power, but thinking about it now (I wrote the function), it should probably check for both in order to qualify.  Also, the 20W is supposed to apply to data equipment per ASHRAE:

```
**computer room**: a room whose primary function is to house equipment for the processing
and storage of electronic data and that has a design electronic data equipment power density
exceeding 20 W/ft2 of conditioned floor area.
```